packages/table/src/table.vue

```diff
@@ -333,32 +391,44 @@
         }
 
         if (this.fit) {
-          this.windowResizeListener = throttle(50, () => {
-            if (this.$ready) this.doLayout();
-          });
-          addResizeListener(this.$el, this.windowResizeListener);
+          addResizeListener(this.$el, this.resizeListener);
         }
       },
 
-      doLayout() {
```

Comment:
This will be a breaking change, as `doLayout` has been suggested as a "fix" to horizontal scrollbar issues: #8488 #6709